runtime/compiler/build/toolcfg/gnu-xlc/common.mk

```diff
@@ -108,6 +108,16 @@ endif
 
 ifeq ($(BUILD_CONFIG),prod)
     CX_DEFINES+=$(CX_DEFINES_PROD)
+    ifeq (,$(findstring ppc64le,$(PLATFORM)))
+         # big endian
+        C_FLAGS+=-qdebug=nscrep
+        # work-around for XL C/C++ 13.1 compiler bug
+        # related to multiple inheritance
+        CXX_FLAGS+=-qdebug=NETHUNK
+    else
+        # little endian
+        CX_FLAGS_PROD+=-qdebug=nscrep
+    endif
```

Comment:
Use of `-qdebug=nscrep` for C++ was previously not conditional on `BUILD_CONFIG`, now it is: why?
Also, that option appears not to have been necessary for C: why is it needed now?

Reply:
Originally `-qdebug=nscrep` applied to both C and C++ flags as `CX_FLAGS_PROD+=$(CX_OPTFLAG) -qdebug=nscrep`. Due to performance isssue, I have applied the new option (`-qdebug=NETHUNK`) only to C++ flags.

Reply:
I see I misread `CX_FLAGS` as `CXX_FLAGS` - thanks for clarifying that.

However, I'm not sure you've answered the other part of my question about why the changes are now only for `BUILD_CONFIG=prod`: surely we don't want the buggy compiler behavior applied for other configurations?

Reply:
For consistency: the `CX_FLAGS_PROD` options applied when `BUILD_CONFIG = prod`. Adding the flag globally degrades the performance. I tried to apply it only where it was required to work-around the compiler bug.

Reply:
@keithc-ca does this resolve your question? See the next line down for how CX_FLAGS_PROD is used.
(or https://github.com/eclipse/openj9/blob/802c34659b3663fe422d352839fcf4867bb9050f/runtime/compiler/build/toolcfg/aix-xlc/common.mk#L110)

Reply:
I hadn't originally noticed that `CX_FLAGS_PROD` was only used if `BUILD_CONFIG` is `prod`, hence my question. I'd like to see `CX_FLAGS_PROD` continue to be defined independent of `BUILD_CONFIG`, at least for consistency with other makefiles.

Reply:
`CX_FLAGS_PROD` is defined independently at /blob/bcb2fff488e7152abc663e3d07f493159327ea01/runtime/compiler/build/toolcfg/gnu-xlc/common.mk#L88.

Reply:
move the content of `else` branch at L89